index.js

```diff
@@ -491,6 +485,10 @@ export class Placement {
    * @returns {Array[string]} Advertising keywords found on the page
    */
   detectKeywords() {
+    // Return previously detected keywords
+    // If this code has already run.
+    if (detectedKeywords.length > 0) return detectedKeywords;
```

Comment:
Will this run every time if the code doesn't detect any keywords? Seems like we maybe want a boolean instead.

Reply:
I ended up doing a `null` vs. list exists (even empty) check.